src/react-app-env.d.ts

```diff
@@ -14,11 +14,15 @@ interface EthereumProviderRequestArguments {
 interface Window {
   ethereum?: {
     isMetaMask?: true
+    isTally?: true
     on?: (...args: any[]) => void
     removeListener?: (...args: any[]) => void
     request?: (args: EthereumProviderRequestArguments) => Promise<unknown>
     isCoinbaseWallet?: boolean
   }
+  tally?: {
```

Comment:
This may be needed in the future for making a custom connector, but at this time I added it for the check to allow compilation.

May not be needed if the check is reduced from `window.tally && window.tally.isTally` to `window.tally`, but I haven't tested that.

Reply:
Alright, thanks for answer! I'm wondering if it's not against yagni and also if creating `window.tally` instead of using `window.ethereum` convention (as it's done e.g. by coinbase or brave wallets) will not cause extra effort to detect and switch between multiple wallets. To be discussed, since it's already not so easy and obvious.

Anyway I've already tested the code and didn't find any bug, looks ok to me ;)